StudentsDB/src/test/java/com/musala/simple/students/db/MainTest.java

```diff
@@ -0,0 +1,145 @@
+package com.musala.simple.students.db;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.musala.simple.students.db.exceptions.SQLConnectionException;
+import com.musala.simple.students.db.exceptions.StudentDataParsingException;
+
+public class MainTest {
+
+	final static Logger LOGGER = LoggerFactory.getLogger(MainTest.class);
+
+	private ClassLoader classLoader = new Main().getClass().getClassLoader();
+	private File file = new File(classLoader.getResource("myData.json").getFile());
+	private final String testPath = file.getAbsolutePath();
+
+	@Test
+	public void testPassingOneArgument() {
+		String[] args = { testPath };
+		try {
+			Main.main(args);
+			assertTrue(true);
+		} catch (StudentDataParsingException | SQLConnectionException e) {
+			e.printStackTrace();
+			assertTrue(false);
```

Comment:
I know it is logically correct to do a `assertTrue(false)` here, but in terms of readability it is bad. I believe there is an `assertFalse` statement provided by jUnit that can be used.

Reply:
Furthermore, I am not sure how correct the assertion is - what are you asserting here? `assertTrue(true)` will always evaluate to true. Better use `verify` to check if some methods within the Main class were called, becuse such assertion seems useless. This seems valid for all tests here.

Reply:
The code is now refactored and the tests for the Main are updated.